Cython/Includes/cpython/complex.pxd

```diff
@@ -12,14 +13,20 @@ cdef extern from "Python.h":
     # PyComplexObject
     # This subtype of PyObject represents a Python complex number object.
 
-    ctypedef class __builtin__.complex [object PyComplexObject]:
+    # It actually isn't even opaque in the limited API - it just isn't present. However, "opaque"
+    # removes enough code that it isn't a compile error
+    ctypedef class __builtin__.complex [object PyComplexObject, check_size opaque_in_limited_api]:
```

Comment:
We're probably not the only ones who provide declarations for builtin types, so rather than adding a new option specifically for the Limited API (which is really a backend, not a frontend thing), we should see if we can exclude all types defined in the `__builtin__` module from the size check when compiling for the Limited API.